workflow/rocoto/gfs_tasks.py

```diff
@@ -2756,6 +2760,8 @@ def efcs(self):
         deps.append(rocoto.add_dependency(dep_dict))
         dep_dict = {'type': 'task', 'name': f'{self.run}_esfc'}
         deps.append(rocoto.add_dependency(dep_dict))
+        dep_dict = {'type': 'task', 'name': f'{self.run.replace("enkf", "")}_ocnanalecen'}
+        deps.append(rocoto.add_dependency(dep_dict))
```

Comment:
I think this needs to be conditional on marine analysis running. As-is it would add a dependency even if we were running without marine turned on at all.

Reply:
Yes, that's the intent @WalterKolczynski-NOAA . Re-centering around the deterministic should happen, with or without the marine DA turned on. 
Is `efcs` used outside of the DA cycling context (atmos or otherwise)?